docs/rules/replace-j-unit4-category-with-jupiter-tag.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Replace JUnit 4 Category with JUnit Jupiter Tag
+ruleId: ReplaceJUnit4CategoryWithJupiterTag
+since: 3.30.0
+minJavaVersion: 8
+remediationCost: 5
+    
+description:
+    This rule offers a stepwise transition to JUnit 5 by replacing JUnit 4 @Category annotations with JUnit Jupiter @Tag annotations.
+tags: ["Java 8", "Testing"]
+---
+
+# Replace JUnit 4 Category with JUnit Jupiter Tag
+
+## Description
+
+This rule replaces each JUnit 4 [@Category](https://junit.org/junit4/javadoc/latest/org/junit/experimental/categories/Category.html) annotation with one or more JUnit Jupiter [@Tag](https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/Tag.html) annotations. This replacements are a further step in direction of a transition from JUnit 4 to JUnit Jupiter.
+
+::: warning Requirements
+This rule requires the following library to be present:
+* junit:junit:4.13
+* org.junit.jupiter:junit-jupiter-api:5.0.0
+
+As for test suites, a refactoring may be necessary.
+For example, 
+```java
+import org.junit.experimental.categories.Categories;
+import org.junit.experimental.categories.Categories.IncludeCategory;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite.SuiteClasses;
+
+import org.test.jsparrow.examples.FirstCategory;
+import org.test.jsparrow.examples.SecondCategory;
+import org.test.jsparrow.examples.ClassATest;
+import org.test.jsparrow.examples.ClassBTest;
+
+@RunWith(Categories.class)
+@IncludeCategory({FirstCategory.class, SecondCategory.class})
+@SuiteClasses({ClassATest.class, ClassBTest.class})
+public class TestSuiteJUnit4 {
+
+}
+```
+must be refactored to
+```java
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.platform.suite.api.IncludeTags;
+import org.junit.platform.suite.api.SelectClasses;
+import org.junit.runner.RunWith;
+
+import org.test.jsparrow.examples.ClassATest;
+import org.test.jsparrow.examples.ClassBTest;
+
+@RunWith(JUnitPlatform.class)
+@IncludeTags({ 
+		"org.test.jsparrow.examples.FirstCategory",
+		"org.test.jsparrow.examples.SecondCategory" })
+@SelectClasses({ClassATest.class, ClassBTest.class})
+public class TestSuiteJupiter {
+}
+```
+Libraries needed by the refactored test suite:
```

Comment:
We should clarify this a little better. Why are those libraries needed by the refactored tests. Are they needed for compiling the code? For running?